### DISCUSSION TITLE
A newbie question

### DISCUSSION THREAD

**Discussion by cyberlauncher** [stashapp/stash#3203] [Q&A] [answered]
Sorry for likely a repeat question but, I'm lost on how to do sorting and classification of videos.
I have a directory structure of /studio_name/series like "../vixen/tushy/," etc.
The file names are structured as an example - "mommysgirl - actress, actress, - (MM-DD-YY).mp4".
As it is now all files are displayed in "Scenes" and stills are in "Images".
How to I get stash to display the files the Movies, Performers, Studios, etc properly?
Thanks in advance.

**Comment by HereIam80** [comment, marked as answer]
While you potentially could parse your directory and file names to match against performers, tags and studios with the settings => tools => scene filename parser, I personally would not work through that approach because you would still miss the cover images, descriptions, titles and whatnot of your scenes.

I recommend working through this guide instead to get the most out of stash:
https://guidelines.stashdb.org/docs/guide-to-scraping/

For now movies are entirely manual work and are just collections of scenes.

**Reply by cyberlauncher** [reply]
Thank you HereIam80! This is what I was looking for.

Would you happen to know how I can have multiple paths to different collections for the data path in Docker?
For example if I have one collection on /alpha/ and another on /beta/

Could I structure the docker-compose.yml like - /dev/disk/by-label/alpha /dev/disk/by-label/beta/:/data  ??

**Reply by JTMWi** [reply]
Couldn't you just create subdirectories in /data with both `- /dev/disk/by-label/alpha:/data/alpha` and `- /dev/disk/by-label/beta:/data/beta`? I wouldn't expect any problems as long as stash has access to the parent dir. You could then add a bash script to create symlinks (in the parent dir) of the files (contained in the subdirs) while ignoring files with existing symlinks and cron that. I could even imagine adding the name of a symlinked file to an ignore list, so that you don't have to read the contents of all the folders every time the script runs.

**Comment by cyberlauncher** [comment]
Yes. Of course.
Thanks!
